webapp/channels/src/components/onboarding_tasklist/onboarding_tasklist.tsx

```diff
@@ -52,10 +52,12 @@ const TaskItems = styled.div`
     transform-origin: left bottom;
     max-height: ${document.documentElement.clientHeight}px;
     overflow-y: auto;
+    display: none;
```

Comment:
do you think would it be worth to set the aria-hidden to the element once is hidden? or would this be sufficient to hide the element from AT?